universe/ValueRefs.cpp

```diff
@@ -98,7 +98,22 @@ namespace {
                           << " target: " << (context.effect_target ? context.effect_target->Name() : "0")
                           << " local c: " << (context.condition_local_candidate ? context.condition_local_candidate->Name() : "0")
                           << " root c: " << (context.condition_root_candidate ? context.condition_root_candidate->Name() : "0")
-                          << "  " << StackTrace();
+                          << " stacktrace: see trace logging";
+            static std::atomic<uint32_t> trace_count = 0;
+            const auto now = std::chrono::duration_cast<std::chrono::minutes>(now.time_since_epoch()).count();
+            static auto ts = std::chrono::duration_cast<std::chrono::minutes>(now.time_since_epoch()).count();
```

Comment:
need to reset `ts` whenever `trace_count` is initially `0` presumably?

Reply:
true